styleguide/derek/_partials/solutions/compare-table.ejs

```diff
@@ -0,0 +1,176 @@
+        </ul>
+      </td>
+      <td class="compareTable-bodyCell">
+        <i class="fa fa-check check-green"></i>
+      </td>
+      <td class="compareTable-bodyCell">
+        <i class="fa fa-check check-green"></i>
+      </td>
+    </tr>
+
+    <tr class="compareTable-bodyRow">
+      <td class="compareTable-bodyCell">
+        <strong>PCI and HIPPA Compliance</strong>
+        <ul class="compareTable-list">
+          <li class="compareTable-listItem">Certified Level 1 Payment Card Industry (PCI) Service Provider on AWS</li>
+          <li class="compareTable-listItem">We can act as a business associate to support customers with HIPAA workloads on AWS</li>
+        </ul>
+      </td>
+      <td class="compareTable-bodyCell">
+        <i class="fa fa-check check-green"></i>
+      </td>
+      <td class="compareTable-bodyCell">
+        <i class="fa fa-check check-green"></i>
+      </td>
+    </tr>
+
+    <tr class="compareTable-bodyRow">
+      <td class="compareTable-bodyCell">
+        <strong>Technical Onboarding Manager (TOM)</strong>
+        <ul class="compareTable-list">
+          <li class="compareTable-listItem">Personal contact to assist with onboarding</li>
+        </ul>
+      </td>
+      <td class="compareTable-bodyCell">
+        <p>Provides initial guidance</p>
+      </td>
+      <td class="compareTable-bodyCell">
+        <p>Coordinates the process of getting your application up and running on AWS</p>
+      </td>
+    </tr>
+
+    <tr class="compareTable-bodyRow">
+      <td class="compareTable-bodyCell">
+        <strong>Best practices and architecture</strong>
+        <ul class="compareTable-list">
+          <li class="compareTable-listItem">A combined set of AWS and Rackspace recommendations from certified AWS architects</li>
+        </ul>
+      </td>
+      <td class="compareTable-bodyCell">
+        <p>Guidance for standard use cases</p>
+      </td>
+      <td class="compareTable-bodyCell">
+        <p>Hands-on design, customized to your specific application</p>
+      </td>
+    </tr>
+
+    <tr class="compareTable-bodyRow">
+      <td class="compareTable-bodyCell">
+        <strong>EC2 operating system management</strong>
+        <ul class="compareTable-list">
+          <li class="compareTable-listItem">Support for Amazon Linux 2015.03+, CentOS 6/7, Ubuntu 12.04/14.04 LTS (Windows coming)</li>
+          <li class="compareTable-listItem">Configuration, optimization, patching, upgrades</li>
+          <li class="compareTable-listItem">Includes core apps like Apache, NGINX, etc.</li>
+        </ul>
+      </td>
+      <td class="compareTable-bodyCell">
+
+      </td>
+      <td class="compareTable-bodyCell">
+        <i class="fa fa-check check-green"></i>
+      </td>
+    </tr>
+
+    <tr class="compareTable-bodyRow">
+      <td class="compareTable-bodyCell">
+        <strong>Monitoring Alarm Ticket Management</strong>
```

Comment:
@amberRrucker the strong was meant for just dummy data. If I add a class for this there wouldn't be any properties in it.